<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>179</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="TotallyAssoCheckBock">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Totally Associative</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="WordsQu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>47</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="WaysQu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>47</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="BlockSiz">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>47</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="WordsLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Words</string>
    </property>
   </widget>
   <widget class="QLabel" name="WaysLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ways</string>
    </property>
   </widget>
   <widget class="QLabel" name="BlockLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Block Size</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>179</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="text">
    <string>Generate</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Generate table&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
